config/tailwind.config.js

```diff
@@ -15,17 +14,105 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        sans: [
-          "Helvetica Neue",
-          "Helvetica",
-          "Arial",
-          ...defaultTheme.fontFamily.sans,
-        ],
+        sans: ['"Titillium Web"', ...defaultTheme.fontFamily.sans],
+        mono: ['"Fira Code"', ...defaultTheme.fontFamily.mono],
+      },
+      fontSize: {
+        lg:   ['1.4375rem', '2.156rem'], /* Base/01 23px, 34.5px */
```

Comment:
Can I ask where these sizes are? Adjusting font size would also mean we need to adjust line height and other typography related values.

Reply:
Copied from the figma foundation design. Line heights are correct so far as I've implemented the design.

Reply:
The second number of each size is the line height from the design.